openpype/hosts/max/plugins/load/load_model_fbx.py

```diff
@@ -13,46 +21,77 @@ class FbxModelLoader(load.LoaderPlugin):
     order = -9
     icon = "code-fork"
     color = "white"
+    postfix = "param"
 
     def load(self, context, name=None, namespace=None, data=None):
         from pymxs import runtime as rt
 
         filepath = os.path.normpath(self.filepath_from_context(context))
         rt.FBXImporterSetParam("Animation", False)
         rt.FBXImporterSetParam("Cameras", False)
+        rt.FBXImporterSetParam("Mode", rt.Name("create"))
         rt.FBXImporterSetParam("Preserveinstances", True)
         rt.importFile(filepath, rt.name("noPrompt"), using=rt.FBXIMP)
 
-        container = rt.GetNodeByName(name)
-        if not container:
-            container = rt.Container()
-            container.name = name
+        namespace = unique_namespace(
+            name + "_",
+            suffix="_",
+        )
+        container = rt.container(
+            name=f"{namespace}:{name}_{self.postfix}")
+        selections = rt.GetCurrentSelection()
+        import_custom_attribute_data(container, selections)
 
-        for selection in rt.GetCurrentSelection():
+        for selection in selections:
             selection.Parent = container
+            selection.name = f"{namespace}:{selection.name}"
 
         return containerise(
-            name, [container], context, loader=self.__class__.__name__
+            name, [container], context,
+            namespace, loader=self.__class__.__name__
         )
 
     def update(self, container, representation):
         from pymxs import runtime as rt
         path = get_representation_path(representation)
-        node = rt.getNodeByName(container["instance_node"])
-        rt.select(node.Children)
+        node_name = container["instance_node"]
+        node = rt.getNodeByName(node_name)
+        namespace, name = get_namespace(node_name)
+        sub_node_name = f"{namespace}:{name}_{self.postfix}"
+        inst_container = rt.getNodeByName(sub_node_name)
+        rt.Select(inst_container.Children)
+        transform_data = object_transform_set(inst_container.Children)
+        for prev_fbx_obj in rt.selection:
+            if rt.isValidNode(prev_fbx_obj):
+                rt.Delete(prev_fbx_obj)
 
         rt.FBXImporterSetParam("Animation", False)
         rt.FBXImporterSetParam("Cameras", False)
+        rt.FBXImporterSetParam("Mode", rt.Name("merge"))
         rt.FBXImporterSetParam("AxisConversionMethod", True)
-        rt.FBXImporterSetParam("UpAxis", "Y")
         rt.FBXImporterSetParam("Preserveinstances", True)
         rt.importFile(path, rt.name("noPrompt"), using=rt.FBXIMP)
+        current_fbx_objects = rt.GetCurrentSelection()
```

Comment:
I guess this section of the load could live in a  `load/fbx_utils.py` ina later PR, since we use it int he `load_camera_fbx` as well